tests/test_pytest_odoo.py

```diff
@@ -0,0 +1,7 @@
+from pytest_odoo import _find_manifest_path
```

Comment:
Note that I switched to GH actions -> can you rebase please?
Next: introduce pre-commit :)

Reply:
good news